lib/service/servicecfg/app.go

```diff
@@ -155,6 +158,9 @@ func (a *App) CheckAndSetDefaults() error {
 	if a.Name == "" {
 		return trace.BadParameter("missing application name")
 	}
+	if a.MCP != nil && a.MCP.Command != "" {
+		a.URI = types.SchemaMCPStdio
+	}
```

Comment:
To allow overriding URI:

```suggestion
	if a.MCP != nil && a.MCP.Command != "" && a.URI == "" {
		a.URI = types.SchemaMCPStdio
	}
```

Reply:
it would be weird if URI is like https but it has a mcp.command. i will keep it simple and let URI rules for now.